src/tests/SConscript

```diff
@@ -108,7 +108,9 @@ def scons():
     denv.d_program('simple_array', 'simple_array.c', LIBS=libs_client)
     denv.d_program('simple_obj', 'simple_obj.c', LIBS=libs_client)
     denv.d_program('simple_dfs', 'simple_dfs.c', LIBS=libs_client)
-    denv.d_program('daos_mw_fi', 'daos_mw_fi.c', LIBS=libs_client)
+    libs_client.remove('daos_tests')
```

Comment:
I'm not aware of anywhere else in the build where we use .remove() however this is only on a list so should be OK.  The alternative would be moving lines 112-113 to line 107 and then calling libs_client.add('daos_tests') before building the others which I think would be cleaner.